pkg/deviceplugin/lister.go

```diff
@@ -21,10 +21,14 @@ type macvtapConfig struct {
 }
 
 type macvtapLister struct {
+	// NsPath is the path to the namespace the lister operates in.
+	NsPath string
```

Comment:
This is nit-picking, but the comment can be avoided by properly naming the variable - i.e. instead of `NsPath`, something like `mainThreadNetNsPath` or whatever.

Not a must.

Reply:
The fact that it is the main thread namespace is a detail hidden for the lister which only operates in the namespace is told to. The main namespace is resolved in the device plugin's main method. In the tests for example it's a test namespace. Let's try with `NetNsPath`.

Reply:
right, mixed things.